Makefile

```diff
@@ -47,14 +47,16 @@ start:
 dev-server: CONFIG_ENV = development
 dev-server: CALYPSO_ENV = desktop-development
 dev-server: NODE_ENV = development
-dev-server: checks desktop$/config.json
+dev-server: checks
 	@echo "\n\n$(GREEN)+------------------------------------------------+"
 	@echo "|                                                |"
 	@echo "|    Wait for calypso to start the dev server    |"
 	@echo "|       and start the app with \`make dev\`        |"
 	@echo "|                                                |"
 	@echo "+------------------------------------------------+$(RESET)\n\n"
 
+	$(MAKE) desktop$/config.json CONFIG_ENV=$(CONFIG_ENV)
```

Comment:
This fixes the issue, but it goes kind of against `make`'s purpose to call it directly here, losing the [target/dependencies semantics](https://github.com/Automattic/wp-desktop/pull/485/files#diff-b67911656ef5d18c4ae36cb6741b7965L50). Is there any way to pass on env vars to dependent targets?

Reply:
Yes, you are right.
After spending hours on #467 to tackle this, I've decided to use this shortcut to get it to a working state as we (@dmsnell and I) ran in to a couple issues – same you are now seeing on #487. 

I'd however love to see this done right by using makes magic to the fullest!